packit_service/worker/mixin.py

```diff
@@ -242,6 +266,36 @@ def issue(self):
         return self._issue
 
 
+class GetBranches(Protocol):
+    @property
+    @abstractmethod
+    def branches(self) -> List[str]:
+        ...
+
+
+class GetBranchesFromIssueMixin(Config, GetBranches):
+    @property
+    def branches(self) -> List[str]:
+        """Get branches names from an issue comment like the following:
+
+
+        Packit failed on creating pull-requests in dist-git (https://src.fedoraproject.org/rpms/python-teamcity-messages): # noqa
+
+        | dist-git branch | error |
+        | --------------- | ----- |
+        | `f37` | `` |
+
+
+        You can retrigger the update by adding a comment (`/packit propose-downstream`) into this issue.
+        """
+        branches = []
+        issue = self.data.project.get_issue(self.data.issue_id)
+        for line in issue.description.splitlines():
+            if m := re.match(r"\s*\| `(\S+)` \|", line):
```

Comment:
I am a little confused, the comment in the docstring is for propose-downstream, but the `GetBranchesFromIssueMixin` is used for triggering Bodhi updates and Koji builds...and for them, the format of the error messages is different if I am checking it correctly :thinking:

Reply:
yes, that should work, thanks!